.github/workflows/ci.yml

```diff
@@ -16,5 +16,9 @@ jobs:
       - name: build
         run: |
           yarn install --frozen-lockfile
-          yarn validate
+          yarn validate:ci
           yarn ci:build
```

Comment:
Aside: we should probably rename this script to `build:ci`

Reply:
Yeah, that'd be good. I'll fire another PR over with that in.